plugins/history_plugin/history_plugin.cpp

```diff
@@ -340,6 +346,10 @@ namespace eosio {
             }
          }
 
+        if( options.count( "history-plugin-prune-timeout-us" )) {
+            prune_timeout = options.at( "history-plugin-prune-timeout-us" ).as<int32_t>();
```

Comment:
Change to `history-api-timeout-us` and also use for value in `get_actions` instead of hard-coded `100000`

Reply:
Rename `prune_timeout` to `history_api_timeout_us` and make it a `fc::microseconds` instead of `int32_t`